examples/runqslower/src/main.rs

```diff
@@ -8,6 +8,7 @@ use libbpf_rs::PerfBufferBuilder;
 use plain::Plain;
 use structopt::StructOpt;
 
+#[path = "bpf/.output/runqslower.skel.rs"]
 mod bpf;
```

Comment:
hm.. I hope user can also control`mod bpf` name, but I'm obnoxious enough already, so let's land this for now.

Reply:
They should be able to. Here the `#[path]` decorator here is overloading the path for `mod bpf`, but in principle, you could name the generated skeleton whatever you like and the module name would correspond.